README.md

```diff
@@ -0,0 +1,84 @@
+# Teapot Server
+
+A simple HTTP server that implements RFC 2324 (Hyper Text Coffee Pot Control Protocol) by returning HTTP 418 "I'm a teapot" status code.
+
+## Implementations
+
+This project provides multiple implementations:
+
+### Hono (Node.js/TypeScript/Bun/Deno)
```

Comment:
Simplify this title. Just note the Language (TypeScript). No need to mention Node.js/Bun/Deno here.